nats/js/kv.py

```diff
@@ -359,6 +359,48 @@ async def keys(self, **kwargs) -> List[str]:
 
         return keys
 
+    async def keys_with_filters(self, filters: List[str] = None, **kwargs) -> List[str]:
```

Comment:
seems we can change this to be a param to `keys()` instead? 
```
keys(filters=[...])
```
no filter means just using `keys()`

Reply:
@wallyqs I followed the convention as mentioned in this [issue](https://github.com/nats-io/nats.py/issues/575).
Should I change it as you mentioned?

Reply:
@wallyqs I followed the convention as mentioned in this [issue](https://github.com/nats-io/nats.py/issues/575).
Should I change it as you mentioned?

Reply:
yes I think for the nats.py client that has access to passing options that would be better